docs/guide/http_cache.md

```diff
@@ -486,96 +459,280 @@ fos_http_cache:
+tagged properly. You may also see `502 Headers too long` errors, and webserver refusing to serve the page.
 
-#### For responses with X-Location-Id
+Here are some options on how to solve this issue:
 
-For custom or eZ Platform controllers still using `X-Location-Id`, a dedicated response listener
-`XLocationIdResponseSubscriber` handles translating this to tags so the cache can be properly invalidated by this bundle.
+**A. Configure to allow larger header:**
 
-!!! note
+*Varnish* config:
+- [http_resp_hdr_len](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-hdr-len) (default 8k, change to i.e. 32k)
+- [http_max_hdr](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-max-hdr) (default 64, change to i.e. 128)
+- [http_resp_size](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-size) (default 23k, change to i.e. 96k)
+- [workspace_backend](https://varnish-cache.org/docs/6.0/reference/varnishd.html#workspace-backend) (default 64k, change to i.e. 128k)
+- Also to see these headers in the `varnishlog`, adapt [vsl_reclen](https://varnish-cache.org/docs/6.0/reference/varnishd.html#vsl-reclen)
 
-    This is currently marked as deprecated. For rendering content it is advised to refactor to use Content View.
-    For other needs there is an FOS tag handler for Twig and PHP that can be used.
+*Nginx* has a default limit of 4k/8k when buffering responses:
+- For [PHP-FPM](https://www.php.net/manual/en/install.fpm.php) setup using proxy module, configure [proxy_buffer_size](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size)
+- For FastCGI setup using fastcgi module, configure [fastcgi_buffer_size](https://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_buffer_size)
 
-#### For custom needs with FOSHttpCache (tagging Relations and more)
+*Fastly* has a `Surrogate-Key` header limit of *16kb*, this can *not* be changed.
 
-For custom needs, including template logic for eZ Platform's Content Relations which is here used as an example, there are two ways
-to tag your responses.
+*Apache* has a [hard](https://github.com/apache/httpd/blob/5f32ea94af5f1e7ea68d6fca58f0ac2478cc18c5/server/util_script.c#L495) [coded](https://github.com/apache/httpd/blob/7e2d26eac309b2d79e467ef586526c10e0f226f8/include/httpd.h#L299-L303) limit of 8kb, so if you face this issue consider using nginx instead.
 
-##### In Twig
+**B. Limit tags header output by system:**
 
-In Twig, you can make sure a response is tagged correctly by using the following Twig operator in your template:
+(1) For inline rendering just displaying the content name, image attribute, and/or link, then it would be enough to:
+- Look into how many inline _(non ESI)_ render calls for content rendering you are doing, and see if you can organize it differently.
+- Consider to inline views not uses other places in template and [tag response in Twig](#Response-tagging-in-Twig) with "relation" tags.
```

Comment:
Not sure what that means "inline views not uses other places"

Reply:
Maybe more like this?


```suggestion
    - For example: Consider to inline the views not used other places in the given template and [tag response in Twig](#Response-tagging-in-Twig) with "relation" tags.
```